From 0cd72923309bbf62e48be7e8f14f7d13e957c61a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 30 Apr 2012 13:55:48 +0000
Subject: [PATCH 1/5] diff: move TRANSLATORS comments so they're picked up
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These TRANSLATORS comments weren't actually picked up by xgettext, the
comment needs to be right before the Q_() function call, evidently it
isn't picked up if that call is wrapped in another function call.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 diff.c |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/diff.c b/diff.c
index 7da16c9..5d06375 100644
--- a/diff.c
+++ b/diff.c
@@ -1413,22 +1413,22 @@ int print_stat_summary(FILE *fp, int files, int insertions, int deletions)
 	 * but nothing about added/removed lines? Is this a bug in Git?").
 	 */
 	if (insertions || deletions == 0) {
-		/*
-		 * TRANSLATORS: "+" in (+) is a line addition marker;
-		 * do not translate it.
-		 */
 		strbuf_addf(&sb,
+			    /*
+			     * TRANSLATORS: "+" in (+) is a line addition marker;
+			     * do not translate it.
+			     */
 			    Q_(", %d insertion(+)", ", %d insertions(+)",
 			       insertions),
 			    insertions);
 	}
 
 	if (deletions || insertions == 0) {
-		/*
-		 * TRANSLATORS: "-" in (-) is a line removal marker;
-		 * do not translate it.
-		 */
 		strbuf_addf(&sb,
+			    /*
+			     * TRANSLATORS: "-" in (-) is a line removal marker;
+			     * do not translate it.
+			     */
 			    Q_(", %d deletion(-)", ", %d deletions(-)",
 			       deletions),
 			    deletions);
-- 
1.7.10.546.gbaa1a.dirty

